common/utils.js

```diff
@@ -3113,16 +3113,23 @@
 
         /**
          * Will return a promise that is resolved when the setup is done
+         *
+         * NOTE: should only be called by config.js (or equivalent configuration app)
          */
-        function setupHead() {
-            addPolyfills();
-            addCanonicalTag();
-            addTitle();
-            setWindowName();
-            overrideDownloadClickBehavior();
-            overrideExternalLinkBehavior();
-            addBodyClasses();
-            return addCustomCSS();
+        function setupHead(settings) {
+            addPolyfills();                                 // needs to be set for navbar functionality (and other chaise functionality)
+            addBodyClasses();                               // doesn't need to be controlled since it relies on .chaise-body class being present
+            addCanonicalTag();                              // controlled by chaise-config value to turn on/off
```

Comment:
> Is addCanonicalTag for example something that only should be done in chaise apps?
I think there are cases that this is purely controlled by the chaise-config. You have to define a chaise-config property to turn it on so calling the function blindly does nothing without the config property.